src/tenant/tests/test_initialization.py

```diff
@@ -141,6 +141,27 @@ def test_default_message_quest_notifies_owner(self):
         owner = User.objects.filter(username="owner", is_staff=True).first()
         self.assertEqual(message_quest.specific_teacher_to_notify, owner)
 
+    def test_default_tags__quest(self):
+        """ test if intro tag is properly assigned to
+        "Welcome to ByteDeck!" + all quests in the orientation campaign.
+        """
+        q_intro = Quest.objects.filter(tags__name="Intro")
+        self.assertEqual(q_intro.count(), 6)
+        self.assertTrue(q_intro.filter(name="Welcome to ByteDeck!").exists())
+        self.assertTrue(q_intro.filter(name="ByteDeck Class Contract").exists())
+        self.assertTrue(q_intro.filter(name="Create an Avatar").exists())
+        self.assertTrue(q_intro.filter(name="Screenshots").exists())
+        self.assertTrue(q_intro.filter(name="Who owns your creations?").exists())
+        self.assertTrue(q_intro.filter(name="Send your teacher a Message").exists())
+
+    def test_default_tags__badges(self):
```

Comment:
See comment about about naming convention for tests

Reply:
done